src/languageSupport/languages/flux/lsp/worker/flux.worker.ts

```diff
@@ -14,6 +14,34 @@ const respond = (msg, cb) => {
   }
 }
 
+/**
+ * Reasons for this buffer:
+ *    * loading times for the monaco-editor client, versus LSP server
+ *    * the required ordering of bootup-messages btwn these two, + the UI messages.
+ *
+ * Details:
+ *    * First, the loading of workers:
+ *        * we lazy load the monaco-editor, including that lib's own worker, which takes time.
+ *        * concurrently, we also load the LSP server thread, since that takes even longer to come online.
+ *    * Next, these messages must occur in order:
+ *          1. monaco-editor request initializing handshake (a.k.a. mutual capabilities).
+ *          2. UI request for didOpen for variables file (Prelude file).
+ *                * reason for order requirement:
+ *                     * monaco-editor creates new FileID (new model objects) in order
+ *                     * LSP assesses these files in order of the model ID
+ *                     * LSP needs to know variable assignments first. `option v = {foo: bar}` before use of foo.
+ *                * how solved:
+ *                     * after monaco-editor initialize, it calls the ConnectionManager constructor.
+ *                     * constructor() --> make prelude file/model.
+ *          3. monaco-editor request for didOpen file.
+ *                * this is the flux query file.
+ *                * triggered on callback after ConnectionManager is instantiated.
+ *      * since the LSP worker takes awhile to boot up:
+ *            * this buffer ensures that these^^ messages are received by the LSP in order.
```

Comment:
Just seems clearer to have it on a single line.

```suggestion
 *      * since the LSP worker takes a while to boot up, this buffer ensures that the messages are received by the LSP in order.
```

Reply:
done